fh-messaging/test/integrate/test_fh_metrics_man_gen_metrics.js

```diff
@@ -120,6 +132,39 @@ exports.tearDown = function (test, assert) {
   });
 };
 
+
+exports.test_deleteMetricsData = function (test, assert) {
```

Comment:
The test looks good and checks against the function not deleting a collection it shouldn't. We should also be checking to make sure it is deleting a collection older than 30 days.

Reply:
That test is already in place, its deleting the ```fhact_20110101``` collection

Reply:
Ah perfect, I missed this while reviewing the code.